ethcore/res/ethereum/kotti.json

```diff
@@ -1,855 +1,855 @@
 {
-	"name":"Kotti Testnet",
-	"dataDir":"kotti",
-	"engine":{
-		"clique":{
-			"params":{
+	"name": "Kotti Testnet",
+	"dataDir": "kotti",
+	"engine": {
+		"clique": {
+			"params": {
 				"period":15,
 				"epoch":30000
```

Comment:
space here?

Reply:
yes, I don't understand how this wasn't changed.